playbook_delete_inventory_vm.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Delete VM from AWS dynamic inventory
+  hosts: "{{ HOSTS }}"
```

Comment:
Any reason that this should be something different than `all`?  Seems like this variable is not required given `limit` can be applied to specific hosts.  This approach also violates the `syntax-check[specific]` ansible-lint rule.

Reply:
Using `all` is a much higher risk in this playbook since failure to use the limit or use the limit correctly would cause a "fail open" scenario in which all machines in your inventory would be deleted. Using a variable makes the playbook "fail closed" since failure to define the variable or using it incorrectly results in an error. I've found this makes for a safer operation and a playbook that can be more easily re-used in Controller.

Is there a way to ignore the default lint rule or write a custom rule? What are your thoughts?

Reply:
I'd suggest naming the parameter something other than `HOSTS` so that it has to be explicitly set as an extra-var.  It's flagging a lint rule for a reason.  Explicitly providing the var seems to strike the balance between safety and flexibility.